src/EnergyPlus/SingleDuct.cc

```diff
@@ -298,7 +298,7 @@ namespace SingleDuct {
                 SimConstVol(SysNum, FirstHVACIteration, ZoneNum, ZoneNodeNum);
 
             } else if (SELECT_CASE_var == SingleDuctConstVolNoReheat) { // AirTerminal:SingleDuct:ConstantVolume:NoReheat
-                Sys(SysNum).SimConstVolNoReheat(SysNum, FirstHVACIteration, ZoneNum, ZoneNodeNum);
+                Sys(SysNum).SimConstVolNoReheat(SysNum, ZoneNodeNum);
```

Comment:
Drop unused arguments. Would be nice to drop `SysNum` since this is a method on Sys(SysNum) but there are other structs in this whole scheme that require `SysNum`.